arbitrum-docs/bold/bold-gentle-introduction.md

```diff
@@ -0,0 +1,128 @@
+
+![bold-safer-withdrawals](./assets/safer-withdrawals-with-bold.png)
+
+BOLD addresses these challenges head-on by introducing a time limit on the existing rollup protocol for resolving disputes, effectively ensuring that challenges conclude within a 7-day window (this window can be decreased by the DAO). This is possible due to two reasons: (1) BOLD’s design allows for challenges between the honest party and any number of malicious adversaries to happen in parallel, and (2) the use of a time limit that will automatically confirm the honest party’s claims if the challenger fails to respond.
+
+### BOLD brings Arbitrum closer to being recognized as a Stage 2 rollup
+
+Inspired by [Vitalik’s proposed milestones](https://ethereum-magicians.org/t/proposed-milestones-for-rollups-taking-off-training-wheels/11571), the team over at L2Beat has assembled a widely recognized framework for evaluating the development Ethereum Rollups. Both Vitalik and the [L2Beat framework](https://medium.com/l2beat/introducing-stages-a-framework-to-evaluate-rollups-maturity-d290bb22befe) refer to the the final stage of rollup development as *“Stage 2 - No Training Wheels”*. A critical criterion for being considered a Stage 2 rollup is to allow anyone to validate the L2 state and post fraud proofs to Ethereum without restraints. This is considered a key requirement for Stage 2 because it ensures *[“that the system is not controlled by a limited set of entities and instead is subject to the collective scrutiny of the entire community”](https://medium.com/l2beat/introducing-stages-a-framework-to-evaluate-rollups-maturity-d290bb22befe).* 
+
+BOLD enables permissionless validation by allowing *anyone* to challenge incorrect Arbitrum state assertions and therefore unlocks new avenues for participation in securing the network, fostering greater inclusivity and resilience. This is made possible because BOLD guarantees that a single, honest entity staked on the correct Arbitrum state assertion will always win against malicious adversaries. The research and work to bring BOLD to life underscores Arbitrum's commitment to scaling Ethereum without compromising on security. 
+
+![pie-slice](./assets/l2beat-pie-chart.png)
+
+With BOLD at its core, Arbitrum charts a course towards being recognized as a Stage 2 rollup by addressing the currently yellow (above) State Validation wedge in [L2Beat's risk analysis pie chart](https://l2beat.com/scaling/summary). BOLD contributes to a more decentralized, efficient, and robust rollup ecosystem while cementing the Arbitrum stack as the most mature and advanced rollup technology out there. And if that’s not amazing enough, L2 Orbit chains launched with BOLD will have permissionless validation available for them right-out-the-box simply by upgrading their Nitro node software.
```

Comment:
```suggestion
With BOLD at its core, Arbitrum charts a course towards being recognized as a Stage 2 rollup by addressing the currently yellow (above) State Validation wedge in [L2Beat's risk analysis pie chart](https://l2beat.com/scaling/summary). BOLD contributes to a more decentralized, efficient, and robust rollup ecosystem. Additionally, L2 Orbit chains launched with BOLD will have permissionless validation available for them out of the box by upgrading their Nitro node software.
```

> L2 Orbit chains launched with BOLD will have permissionless validation available for them out of the box by upgrading their Nitro node software.

It may be worth addressing two audiences specifically here, assuming my understanding is correct: Current Orbit chain operators (who can enable BOLD by upgrading Nitro), and future Orbit chain operators (who will have BOLD enabled by default by way of Nitro). Right now we're combining these two.

Not sure if we want to address whether this BOLD-enabled-by-default Nitro upgrade will be required, or the impact it will have on chain operators (could it be argued that this is a breaking change, or is it a completely unobtrusive 'pure win' from RaaS providers' perspective).

Reply:
Thanks - great call out. Will address in a separate commit!